kube-runtime/src/watcher.rs

```diff
@@ -192,6 +199,16 @@ pub enum ListSemantic {
     Any,
 }
 
+/// Configurable watcher listwatch semantics
+#[derive(Clone, Default, Debug, PartialEq)]
+pub enum WatcherMode {
```

Comment:
I renamed this one because clippy is unhappy abou the stuttering of package name and type name `watcher::WatcherMode`